### ISSUE TITLE
Only works for FineReader8 schema files

### ISSUE THREAD

**Issue by kba** [kba/AbbyyToAlto#1]
This only works for Abbyy v8 schema compliant files.

@zuphilip @stweil @jze Do you have FineReader 8 XML files so I can confirm correctness of an XSLT port?

**Comment by stweil** [comment]
I'm afraid that all of our ABBYY files use http://www.abbyy.com/FineReader_xml/FineReader10-schema-v1.xml. See http://digi.bib.uni-mannheim.de/fileadmin/digi/429271662/abbyy/ for examples.

**Comment by kba** [comment]
I tried with v10 schema-compliant files but they yielded no sensible results. Thanks for the feedback, though, we could use the Abbyy output as a baseline for a v10 conversion.

**Comment by zuphilip** [comment]
No, I don't have example for that. But maybe you find some examples by google for the namespaces: https://www.google.de/search?q=http://www.abbyy.com/FineReader_xml/FineReader8-schema-v2.xml

**Comment by jtlz2** [comment]
@kba Is this ticket still live? :\ I also tried with v10 schema and got just a header out. FineReader does now output alto though - but I'm trying - and failing - to convert it to hocr so I can use [hocr-tools](https://github.com/tmbdev/hocr-tools) :\

Any ideas on how to do this would be gratefully received...
